Gradata/tests/test_injection_guard.py

```
@@ -0,0 +1,291 @@
+class TestIsSuspiciousLengthCap:
+    """Payload: context-window saturation via excessive input."""
+
+    def test_length_exceeds_cap(self) -> None:
+        long_text = "x" * 100_001
+        suspicious, reason = is_suspicious(long_text)
+        assert suspicious is True
+        assert reason.startswith("length_exceeds_cap")
+
+    def test_length_at_cap_not_suspicious(self) -> None:
+        long_text = "x" * 100_000
+        suspicious, reason = is_suspicious(long_text)
```

Ruff (0.15.13)

[warning] Unpacked variable `reason` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)